docs/ide-usage.md

```diff
@@ -188,11 +179,6 @@ If you do not wish to use the automated setup, you can follow the [manual setup]
    #### Javascript
    Go to `Languages & Frameworks → JavaScript` and select `ECMAScript 6` for the `JavaScript language version`.
 
-   #### Additional Build Tasks
-   1. Open `View → Tool Windows → Gradle`.
-   1. Under `Tasks → intellij idea setup`, look for the task `enhanceIntellijOutputClasses`.
-   1. Right click and select `Execute After Build`.
-
```

Comment:
These would be dead instructions though (like dead code) as they're only applicable to projects set up in the past. I feel they belong in the issue tracker rather than the codebase.